portfolio/src/main/webapp/script.js

```diff
@@ -39,22 +36,22 @@ function toggleDetailsButton() {
   }
 }
 
-// Gets the data from server sides.
-function getServerData() {
+/** Gets the data from server sides. */
+function loadComments() {
     fetch('/data')  
     .then(response => response.json()) 
     .then((data) => {
-        createListElement(data, 'comments');
+        createCommentElement(data, 'comments');
     });
```

Comment:
you should probably handle error as well.

Reply:
alright